content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -224,7 +207,7 @@ Here is some example output of the resulting events you see from running `kubect
 对于 `PostStart`，这是 `FailedPostStartHook` 事件，对于 `PreStop`，这是 `FailedPreStopHook` 事件。
 要自己生成失败的 `FailedPreStopHook` 事件，请修改
 [lifecycle-events.yaml](https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/lifecycle-events.yaml)
-文件将 postStart 命令更改为 ”badcommand“ 并应用它。
+文件将 postStart 命令更改为 "badcommand" 并应用它。
```

Comment:
Discuss: I think the original one is correct, not sure which one is better.

Reply:
Good catch. Changed again.